configurations/bossaso.json

```diff
@@ -4,7 +4,8 @@
   "ActivationFlowNames": [
     "csap_s03w01_activation",
     "csap_s03w02_activation",
-    "csap_s03w03_activation"
+    "csap_s03w03_activation",
+    "csap_s03w04_activation"
```

Comment:
To confirm  ` "csap_s03w05_activation"` is not needed here because bossaso will already have finished?

Reply:
Yes - we're using 'weeks' in TextIt; 'episodes' in the pipeline.